### PR TITLE
chore: update tests to use 1.0 image, enable module-swapper for tests

### PR DESCRIPTION
**Pull request by bharathkkb**
- update images to 1.0
- enable module-swapper before int tests to swap example [upstream refs ](https://github.com/terraform-google-modules/terraform-google-network/blob/master/examples/simple_project/main.tf#L27-L28)in tests
- update note for TF versions
- switch build to init in serial followed by converge in parallel

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_whitespace
 Failed whitespace check. More details below. 
 ```bash 
Checking for trailing whitespace
Checking for missing newline at end of file
Error: No newline at end of file ./README.md
 ```

File: README.md [hunk 1]
```diff
@@ -12,7 +12,12 @@ Sub modules are provided for creating individual vpc, subnets, and routes. See t
 
 ## Compatibility
 
-This module is meant for use with Terraform 0.12. If you haven't [upgraded](https://www.terraform.io/upgrade-guides/0-12.html) and need a Terraform 0.11.x-compatible version of this module, the last released version intended for Terraform 0.11.x is [0.8.0](https://registry.terraform.io/modules/terraform-google-modules/network/google/0.8.0).
+This module is meant for use with Terraform 0.13+ and tested using Terraform 1.0+.
+If you find incompatibilities using Terraform `>=0.13, <1.0`, please open an issue.
```
Comment (morgante) [inline README.md:16]:
```suggestion
If you find incompatibilities using Terraform `>=0.13`, please open an issue.
```

**Review by morgante — DISMISSED** [review]
One nit about compatibility message.

**Comment by bharathkkb** [comment]
Merging as previous LGTM was dismissed by when I committed suggestion

**Pull request merged**